pkg/apis/monitoring/v1/thanos_types.go

```diff
@@ -29,7 +29,7 @@ const (
 // ThanosRuler defines a ThanosRuler deployment.
 // +genclient
 // +k8s:openapi-gen=true
-// +kubebuilder:resource:categories="prometheus-operator"
+// +kubebuilder:resource:categories="prometheus-operator",shortName="thruler"
```

Comment:
```suggestion
// +kubebuilder:resource:categories="prometheus-operator",shortName="ruler"
```

Reply:
Thank you @simonpasquier 
I have addressed it